index.html

```diff
@@ -379,7 +379,7 @@ <h2 class="font-inter font-center contact-para">
           <form
             class="contacts-form d-flex"
             action="https://formspree.io/f/mqkwjeyr"
-            method="POST"
+            method="POST" id="form"
           >
             <label for="name">name:</label>
```

Comment:
- [ ] OPTIONAL
The full name field of your _form_ is **accepting digits** as names. It will therefore be a nice implementation to add validations against your form inputs. Kindly ensure to improve upon this aspect.